recipes/rever/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rever" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "ce8924c4f2feb57f8151bdf8ab002f8fcedebc32855269cffbaa321d5023f352" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ergs/rever/archive/0.1.0.tar.gz
```

Comment:
Maybe use the `version` jinja template variable here instead of hardcoding the version for consistency.